frontend/src/services/Api.ts

```diff
@@ -589,49 +640,50 @@ export const getNodeGraphElements = (node: NodeParamsType, params: any) => {
               node.aggregateValue!,
               node.service
             ),
-            params,
+            queryParams,
             {}
           );
     case NodeType.APP:
     case NodeType.BOX: // we only support app box node graphs, so treat like app
       return newRequest<GraphDefinition>(
         HTTP_VERBS.GET,
         urls.appGraphElements(node.namespace.name, node.app, node.version),
-        params,
+        queryParams,
         {}
       );
     case NodeType.SERVICE:
       return newRequest<GraphDefinition>(
         HTTP_VERBS.GET,
         urls.serviceGraphElements(node.namespace.name, node.service),
-        params,
+        queryParams,
         {}
       );
     case NodeType.WORKLOAD:
       return newRequest<GraphDefinition>(
         HTTP_VERBS.GET,
         urls.workloadGraphElements(node.namespace.name, node.workload),
-        params,
+        queryParams,
         {}
       );
     default:
       // default to namespace graph
-      return getGraphElements({ namespaces: node.namespace.name, ...params });
+      return getGraphElements({ ...params, namespaces: node.namespace.name });
```

Comment:
getGraphElements does not accept 'cluster' as a second param?

Reply:
The graph does not limit to a specific cluster.  Only namespaces.  Graph will always query across cluster, to limit to a specific cluster the user can narrow things in the UI.